src/main/cpp/subsystems/Arm.cpp

```diff
@@ -69,37 +102,22 @@ void Arm::Periodic() {
             VOKC_CALL(this->ManualControl());
             break;
         case Auto:
-            //this->interface_->arm_lift_power = this->arm_pid_->Caluculate(this->interface_->arm_encoder);
-            //this->interface_->arm_up_power = this->arm_pid_->Caluculate(this->interface_->arm_extend_encoder);
+            VOKC_CHECK(interface_ != nullptr);
+            VOKC_CHECK(this->arm_pid_ != nullptr);
+            this->interface_->arm_lift_power = this->arm_pid_->Calculate(this->interface_->arm_duty_cycle_encoder);
+            this->interface_->arm_extend_power = this->inches_pid_->Calculate(this->interface_->arm_extend_encoder);
```

Comment:
It would be best to make this block of code a separate function so the switch-case is clean

Reply:
like AutoArm() or something like we have for Manual? . . . yeah definitely